shelley/chain-and-ledger/formal-spec/chain.tex

```diff
@@ -450,14 +467,15 @@ \subsection{Update Nonce Transition}
         \eta_0 & \Seed & \text{epoch nonce} \\
         \eta_c & \Seed & \text{candidate nonce} \\
         \eta_v & \Seed & \text{evolving nonce} \\
+        \eta_h & \Seed & \text{hash from end of previous epoch} \\
```

Comment:
not necessarily a hash, right? maybe "seed generated from hash of previous epoch" ?

Reply:
yes, true, it's a hash turned into a nonce :) it's hard to explain concisely...

Reply:
ok, I've used you description in all three spots.thanks!